prog/weaver/main.go

```diff
@@ -245,7 +248,7 @@ func main() {
 		networkConfig.PacketLogging = nopPacketLogging{}
 	}
 
-	overlay, bridge := createOverlay(datapathName, ifaceName, config.Host, config.Port, bufSzMB)
+	overlay, bridge := createOverlay(datapathName, ifaceName, config.Host, config.Port, bufSzMB, useAWSVPC)
```

Comment:
seems simpler just to put the conditional on `useAWSVPC` here, so you leave `createOverlay()` alone.